src/components/ui/Shell/Shell.tsx

```diff
@@ -0,0 +1,168 @@
+import { cva, VariantProps } from "class-variance-authority";
+import ReactMarkdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+
+const shellStylesOuter = cva(
+  "fixed inset-0 mx-auto mx-auto flex w-full h-full max-w-4xl border-solid border-2 border-zinc-200 bg-white dark:bg-zinc-900 justify-center rounded-lg px-4 px-4 sm:px-6 sm:px-6 md:px-8 md:px-8"
+);
+
+const shellStylesInner = cva(
```

Comment:
If we're not using any variants, cva is just extra overhead - let's throw these inline